src/groups/array_manifold.jl

```diff
@@ -0,0 +1,154 @@
+array_value(e::Identity) = e
```

Comment:
This overall is a nice addition, since it does the same as we propose for the array manifold anyways – proper types and more tests during computations.